### PR TITLE
Tools 1887

### PR DESCRIPTION
**Pull request by voyenavion**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,8 +20,8 @@ MAINTAINER cronic@horizen.io
 
 SHELL ["/bin/bash", "-c"]
 
-# Latest release zen 3.1.0
-ARG ZEN_COMMITTISH=v3.1.0
+# Latest release zen 3.2.0
```
Comment (yurikoinaba) [inline Dockerfile:23]: `3.2.1`

**Review by yurikoinaba — COMMENTED** [review]
Not sure why `development` and `rosetta-zen` (the default branch) are not in sync.
I'd say make the changes off of the `rosetta-zen` branch bc that is on the latest Zend version.
@voyenavion

**Comment by yurikoinaba** [comment]
Could you also take a look at the failing checks?

**Comment by voyenavion** [comment]
merged same change from a different pr

**Pull request closed without merging**